src/migrations/20171129064057_multicontainer.js

```diff
@@ -0,0 +1,311 @@
+
+var defaultLegacyVolume = function (appId) {
+	return `resin-app-${appId}`
+}
+
+var tryParse = function (obj) {
+	try {
+		return JSON.parse(obj)
+	}
+	catch(e) {
+		return {}
+	}
+}
+
+var singleToMulticontainerApp = function (app) {
+	// From *very* old supervisors, env or config may be null
+	// so we ignore errors parsing them
+	let conf = tryParse(app.config)
+	let env = tryParse(app.env)
+	let environment = {}
+	let appId = parseInt(app.appId)
+	for (let key in env) {
+		if (!/^RESIN_/.test(key)) {
+			environment[key] = env[key]
+		}
+	}
+	let newApp = {
+		appId: appId,
+		commit: app.commit,
+		name: app.name,
+		releaseId: 1,
+		networks: {},
+		volumes: {}
+	}
+	let defaultVolume = defaultLegacyVolume(appId)
+	newApp.volumes[defaultVolume] = {}
+	let updateStrategy = conf['RESIN_SUPERVISOR_UPDATE_STRATEGY']
```

Comment:
These can be done as
```js
const updateStrategy = _.get(conf, 'RESIN_SUPERVISOR_UPDATE_STRATEGY', 'download-then-kill')
```
etc

Reply:
We don't have lodash here (or any other library, for that matter)